Content.Client/Power/PowerMonitoring/PowerMonitoringSystem.cs

```diff
@@ -0,0 +1,153 @@
+using Content.Client.Stylesheets;
+using Content.Shared.Power;
+using JetBrains.Annotations;
+using Robust.Client.Graphics;
+using Robust.Client.UserInterface.Controls;
+using System.Linq;
+using static Robust.Client.UserInterface.Control;
+
+namespace Content.Client.Power.PowerMonitoring;
+
+[UsedImplicitly]
+public sealed class PowerMonitoringSystem : EntitySystem
```

Comment:
This is super not allowed, system methods shouldnt ever care about UI controls, move these to the window or make a UI controller if necessary